### PR TITLE
Bookstore: create and remove books

### PR DESCRIPTION
**Pull request by david-lafontant**
- Configure Redux Store and write actions and reducer for adding and removing books. 
- Add some UI elements to your components and make them dispatch actions. NOTE: editing an existing book and updating the progress is not part of this step. Neither is the styling of the application's components (set title and author - NOTE: categories and comments are NOT part of this step).
- The project should follow the [Project Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_create_and_remove_book.md#project-requirements)

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -1,20 +1,76 @@
...
+
+  const submitBookToStore = (event) => {
+    event.preventDefault();
+    document.querySelector('form').reset();
+
+    const newBook = {
+      id: uuidv4(),
+      title: formState.title[0],
+      author: formState.author[0],
+      category: formState.author[0],
```
Comment (TedLivist) [inline src/components/Form.js]: - [ ] I think you meant to have `category: formState.category[0],` here. Kindly fix that.

File: src/components/Form.js [hunk 2]
```diff
@@ -1,20 +1,76 @@
...
+          onChange={handleChange}
+          required
+        >
+          <option value="">Select a category</option>
+          <option value="Category 1">Novel</option>
+          <option value="Category 2">Essay</option>
+          <option value="Category 3">History</option>
+          <option value="Category 2">Biography</option>
+          <option value="Category 3">LItterature</option>
+        </select>
```
Comment (TedLivist) [inline src/components/Form.js]: - [ ] I would recommend you change these **`value`** attributes to something more meaningful to represent the actual select option like so:

  lafontant-select

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @david-lafontant,

Good job so far!
✔️ Books can be added and removed
✔️ Form uses local state
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] **In your README, kindly add detailed instructions on how to set up and run your app locally i.e. cloning to `npm install`, etc**

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Form.js (hunk 1 above)
Comment (david-lafontant) [reply src/components/Form.js]: Asked change implemented

File: src/components/Form.js (hunk 2 above)
Comment (david-lafontant) [reply src/components/Form.js]: Asked change implemented

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @david-lafontant 👋🏻, 

Great job implementing the changes requested 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**